cinema/serializers.py

```diff
@@ -14,10 +14,10 @@ def create(self, validated_data):
 
     def update(self, instance, validated_data):
         instance.title = validated_data.get("title", instance.title)
-        instance.description = validated_data.get(
-            "description", instance.description
-        )
-        instance.duration = validated_data.get("duration", instance.duration)
+        instance.description = (
+            validated_data.get("description", instance.description))
```

Comment:
PEP8 allows several variations on the use of parentheses but the mostly used nowadays and applied by the most popular formatters is smth like this. I'd recommend to stick with it.

```
def example(session):
    result = (
        session.query(models.Customer.id)
        .filter(
            models.Customer.account_id == account_id,
            models.Customer.email == email_address,
        )
        .order_by(models.Customer.id.asc())
        .all()
    )
  ```
  
  See also what the most popular formatter is saying about this:
  https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html